### PR TITLE
acl2: Update to 8.0

### PR DESCRIPTION
**Pull request by JacksonIsaac**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.4 17E199
Xcode 9.2 9C40b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by pmetzger** [comment]
This Portfile is so complicated I don't feel qualified to check it on my own. I'd like someone else to also give it an eyeball.

Also, I would suggest that given how much you've done here, perhaps you should adopt the port as the maintainer?

File: math/acl2/Portfile [hunk 1]
```diff
@@ -1,57 +1,60 @@
...
+    a specification language, a modeling language, \
+    a formal mathematical logic, or a semi-automatic \
+    theorem prover. Because the meta-language is the same \
+    as the language (a subset of Common Lisp), it is very \
+    flexible.
+
+# user_notes Users who want to use ACL2 for serious work should
+# install the certify variant (sudo port install +certify),
+# which will certify (i.e., prove all of the theorems)
+# in the included examples. This can take several hours.
```
Comment (raimue) [inline math/acl2/Portfile]: Was this supposed to end up in `notes` at some point in the past?

Comment (JacksonIsaac) [reply math/acl2/Portfile]: I am not sure exactly what the author wanted to do here.

+certify and +regression takes a lot of time to build (got stuck for me when my mac went to sleep and didn't resume back). These two variants also fails during trace mode for some reason; neverpanic and I were discussing on IRC and couldn't find out the reason.

Something to do with /usr/bin/perl getting replaced by /usr/bin/perl5.18 in sip_workaround folder and throwing not found error.

Most likely we could remove these two variants but some users might want to use +certify. I have tested other variants and also the combinations of them and everything works fine. Atleast the basic functionalities lg2m.

Comment (pmetzger) [reply math/acl2/Portfile]: Fix it as you prefer and let me know (or merge this yourself). I still think you should add yourself as the maintainer.

Comment (JacksonIsaac) [reply math/acl2/Portfile]: Ok sure. Thanks :) I'll make the changes regarding the notes and merge it.

File: math/acl2/Portfile [hunk 2]
```diff
@@ -32,10 +32,11 @@ long_description    \
     as the language (a subset of Common Lisp), it is very \
     flexible.
 
-# user_notes Users who want to use ACL2 for serious work should
-# install the certify variant (sudo port install +certify),
-# which will certify (i.e., prove all of the theorems)
-# in the included examples. This can take several hours.
+notes "Users who want to use ACL2 for serious work should
```
Comment (raimue) [inline math/acl2/Portfile]: The other uses of `notes` in this Portfile need to be changed to `notes-append`, or they will overwrite this.

Comment (JacksonIsaac) [reply math/acl2/Portfile]: Thanks for pointing this out :) Fixed it now. Will keep these things in mind in future.

**Comment by ryandesign** [comment]
Is there a reason why you did not use the release download available at https://github.com/acl2-devel/acl2-devel/releases?

**Comment by ryandesign** [comment]
No; please don't override the github portgroup's `master_sites` unless the distfile is not hosted at GitHub. I mean: why didn't you use `github.tarball_from releases` and adjust the checksums accordingly? Consult the [github portgroup documentation](https://guide.macports.org/#reference.portgroup.github).

**Comment by JacksonIsaac** [comment]
Okay, sure. I wasn't aware of this option. Thanks for pointing this out :)

**Pull request merged**